ui/components/Editor/EditorMerged/styles.ts

```diff
@@ -1,6 +1,12 @@
 import styled from "styled-components";
 import { ScaleInboxIcon } from "@hoory/shared-assets";
-import { ifProp, theme, prop, inLessThan } from "../../../../helpers";
+import {
+  ifProp,
+  theme,
+  prop,
+  inLessThan,
+  makeRgbaFromTheme,
+} from "../../../../helpers";
```

Comment:
@grigormargaryan jan, use `styled-components-helpers`, we will remove this from the common frontend
```suggestion
} from "styled-components-helpers";
```